docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -114,33 +148,38 @@ Play is being rebuilt and progressively rolled out to more users. This section e
+When opening Play in Self-Managed, you will be asked to fill in the details of your cluster.
 
-- Play runs on community-built projects, as described in the [Zeebe-Play repository](https://github.com/camunda-community-hub/zeebe-play).
+Find information below, as well as example values for the simple docker setup.
 
-- Play is run on completely isolated Camunda-hosted infrastructure from the core SaaS or Self-Managed Camunda deployment. It can only receive information from Web Modeler and can only communicate externally using Connectors and the user-defined secrets in the secret store or BPMN diagram.
+![play cluster config](img/play-cluster-configuration.png)
 
-  :::note
-  Play is not authenticated, so anyone with the URL can access it. You should not submit personal or confidential information to Play.
-  :::
+| Name              | Description                                  | Example value                                                                     |
+| ----------------- | -------------------------------------------- | --------------------------------------------------------------------------------- |
+| Cluster endpoint  | Address where your cluster can be reached.   | `http://zeebe:26500`                                                              |
+| Operate base url  | Address where Operate can be reached.        | `http://operate:8080`                                                             |
+| Operate audience  | Permission name for Operate                  | `operate-api`                                                                     |
+| Tasklist base url | Address where Tasklist can be reached.       | `http://tasklist:8080`                                                            |
+| Tasklist audience | Permission name for Tasklist                 | `tasklist-api`                                                                    |
+| Zeebe rest url    | Address where Zeebe rest api can be reached. | `http://zeebe:8080`                                                               |
+| Client id         | Name of your registered client               | `zeebe`                                                                           |
+| Client secret     | Password for your registered client          | `zecret`                                                                          |
+| OAuth token url   | Token issuer server                          | `http://keycloak:8080/auth/realms/camunda-platform/protocol/openid-connect/token` |
+| OAuth audience    | Permission name for Zeebe                    | `zeebe-api`                                                                       |
 
-- To mitigate this risk, a 34-character randomly generated UUID is in the URL. Each session lasts approximately 20 minutes, and the Play environment and its data are automatically deleted at the end of each session. You can reset the session timer by re-opening Play.
+### Simple docker setup
```

Comment:
For the docs to follow the user journey, the setup should be before configuration